Makefile

```diff
@@ -21,14 +21,6 @@ all: openconfig_public protos validate_paths
 openconfig_public:
 	tools/clone_oc_public.sh openconfig_public
 
-validate_paths: openconfig_public proto/feature_go_proto/feature.pb.go
-	go run -v tools/validate_paths.go \
```

Comment:
`tools/validate_paths.go` can also be removed since this tool wouldn't be used any more.

Reply:
Done